src/coralogix/schema.js

```diff
@@ -69,7 +69,7 @@ const schema = {
         ),
       },
       request: {
-        id: str(vcl`req.http.X-CDN-Request-ID`),
+        id: str(vcl`if(req.http.X-CDN-Request-ID, req.http.X-CDN-Request-ID, randomstr(8, "0123456789abcdef") + "-" + randomstr(4, "0123456789abcdef") + "-" + randomstr(4, "0123456789abcdef") + "-" + randomstr(1, "89ab") + randomstr(3, "0123456789abcdef") + "-" + randomstr(12, "0123456789abcdef"))`),
```

Comment:
What's the point of generating a request ID just for the log? It doesn't serve any purpose.

Reply:
1. increases the uniformity of logs
2. makes it possible to correlate logs between multiple targets
3. makes it posible to count something when grouping